.kitchen.yml

```diff
@@ -9,13 +9,68 @@ platforms:
   - name: ubuntu-14.04
     run_list:
     - "recipe[apt::default]"
+  - name: centos-7.1-vbox
```

Comment:
I would give up one of my cats if the Chef folks provided that conditional depends functionality ;)